consumer_kafka/src/test/resources/application-test.yml

```diff
@@ -1,8 +1,6 @@
 spring:
   kafka:
     bootstrap-servers: ${spring.embedded.kafka.brokers}
-    consumer:
```

Comment:
The test does not need any consumer properties - the application configures that. However, it doe need the producer props so that the stubrunner can send the test messages.